docs/modules/ROOT/pages/sentinel.adoc

```diff
@@ -653,6 +653,102 @@ exports.handler = async function(params) {
 }
 ----
 
+[[customizing-notification-messages]]
+== Customizing Notification Messages
+
+You can optionally modify the notification message content and formatting using the checkbox below the notification channel selector.
+
+=== Message Syntax
+
+Custom notifications support a limited set of markdown syntax:
+
+* Bold
+* Italic
+* Links
+
+There is partial support for additional markdown syntax, but rendering behavior varies by platform.
+
+=== Dynamic Content
+
+Custom notification templates render dynamic content using inline templating. Any string surrounded by double curly braces will be resolved against the <<Event Schema>>. Deeply nested items (including those in arrays) can be accessed using dot notation.
+
+In addition to the standard event schema, the following parameters are injected for usage in custom notification messages: 
+
+* `transaction.link`
+* `matchReasonsFormatted`
+
+=== Character Limit
+
+Messages will be truncated if they exceed a platform's character limit. The best practice is to limit messages to 1900 characters.
+
+=== Example
```

Comment:
Open for discussion, but I'd make the Example section the first one after the introductory sentence ("You can optionally modify the notification message content and formatting using the checkbox below the notification channel selector(...)").

The detailed explanations of syntax and dynamic content will make much more sense after skimming the example

Reply:
makes sense, done